.github/workflows/canary-integration-test.yml

```diff
@@ -1398,6 +1398,71 @@ jobs:
         run: |
           timeout 60 sh -c 'until [ "$(kubectl get cm -n rook-ceph rook-ceph-csi-mapping-config  -o jsonpath='{.data.csi-mapping-config-json}' | grep -c "rook-ceph-secondary")" -eq 1 ]; do echo "waiting for rook-ceph-csi-mapping-config to be created with cluster ID mappings" && sleep 1; done'
 
+      - name: replace operator and cluster images for rados namespace mirroring
+        run: |
+          kubectl -n rook-ceph patch deployment rook-ceph-operator --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value": "quay.io/paarora/rook-ceph:v614"}]'
+          kubectl -n rook-ceph patch cephcluster my-cluster --type='merge' -p '{"spec": {"cephVersion": {"image": "quay.io/paarora/rook-ceph:v609"}}}'
+          kubectl -n rook-ceph-secondary patch cephcluster my-cluster --type='merge' -p '{"spec": {"cephVersion": {"image": "quay.io/paarora/rook-ceph:v609"}}}'
+          sleep 10
+
+      - name: create mirrored rados namespace on cluster 1
```

Comment:
Whenever there are configurations of two namespaces, I'd recommend the pattern:
- Update namespace 1
- Update namespace 2
- Wait for namespace 1 config
- Wait for namespace 2 config
This way, the namespaces can reconcile in parallel an the test is a little more efficient.